test/unit/eval_algorithms/test_classification_accuracy.py

```diff
@@ -77,6 +78,15 @@
     EvalScore(name=BALANCED_ACCURACY_SCORE, value=1 / 2),
     EvalScore(name=PRECISION_SCORE, value=2 / 3),
     EvalScore(name=RECALL_SCORE, value=2 / 3),
+    EvalScore(name=UNKNOWN_RATIO, value=0.0),
+]
+
+DATASET_SCORES_WO_CONFIG = [
```

Comment:
Can you explain what this represents? I see that you replaced all instances of `DATASET_SCORES` with `DATASET_SCORES_WO_CONFIG`. Shouldn't we still have some tests that use `DATASET_SCORES`? Also, could you please come up with a case where the expected precision and recall scores are different?